PipelinesCD.yaml

```
@@ -34,7 +34,7 @@ resources:
   - downloadBuild: WebApp  
```

Dont use patterns ~!@#$%^&*()_+=-}{][":;',./<>?

---

Done

---

Looks good to me! @ #48 
```suggestion
  - downloadBuild: WebApp  ### 
```